_posts/2015-05-14-function-bind.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "Function Bind Syntax"
+author: James Kyle
+date:   2015-05-14 19:30:00
+categories: announcements
+share_text: "New in Babel 5.4: Function Bind Syntax"
+---
+
+Babel 5.4 was just released and with it comes support for a new
+[experimental ES7 syntax](https://github.com/zenparsing/es-function-bind) designed by
+Kevin Smith ([@zenparsing](https://github.com/zenparsing)) and implemented in
+Babel by Ingvar Stepanyan ([@RReverser](https://github.com/RReverser)).
+
+> ***Warning: This syntax is highly experimental and you should not use
+> it for anything serious (yet).***
```

Comment:
Sure, should I send them to https://github.com/zenparsing/es-function-bind/issues?